physical/distinct.go

```diff
@@ -4,20 +4,21 @@ import (
 	"context"
 
 	"github.com/cube2222/octosql/execution"
+	"github.com/cube2222/octosql/physical/metadata"
 	"github.com/pkg/errors"
 )
 
 type Distinct struct {
-	Child Node
+	Source Node
```

Comment:
I understand that this change is purely stylistic/to improve naming?

Reply:
Yea, to make the naming consistent.